docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -0,0 +1,403 @@
+    
+    Possible values: `anomaly_score`, `yhat`, `yhat_lower`, `yhat_upper`, `y`. You can choose all or some of them.
+
+* `reader`
+  * `class` - defining where to read data from. Here `"reader.vm.VmReader"`
+  * `datasource_url` - Data source. An HTTP endpoint that serves `/api/v1/query_range`.
+  * `queries`: - MetricsQL (extension of PromQL) expressions,  where you want to find anomalies.
+
+  You can put several queries in a form:
+  `<QUERY_ALIAS>: "QUERY"`. QUERY_ALIAS will be used as a `_key` label in generated metrics and anomaly scores.
+
+* `writer`
+    * `class` - defining where to write data. Here `"writer.vm.VmWriter"`
+    * `datasource_url` - Output destination. An HTTP endpoint that serves `/api/v1/import`.
+
+_____________________________________________
+## VmAnomaly output
+As the result of running VmAnomaly, it produces the following metrics:
+- `anomaly_score` - the main one. Ideally, if it is between 0.0 and 1.0 it is considered to be a non-anomalous value. If it is greater than 1.0, it is considered an anomaly (but you can reconfigure that in alerting config, of course), 
+- `yhat` - predicted expected value, 
+- `yhat_lower` - predicted lower boundary, 
+- `yhat_upper` - predicted upper boundary, 
+- `y` - initial query result value.
+
+Here is an example of how output metric will be written into VictoriaMetrics:
+`anomaly_score{for="node_cpu_rate", cpu="0", instance="node_exporter:9100", job="node-exporter", mode="idle"} 0.85`
+
+____________________________________________
+
+## VmAlert congiguration
+Here we provide an example of the config for VmAlert `vmalert_config.yml`.
+```
+groups:
+- name: AnomalyExample
+  rules:
+  - alert: HighAnomalyScore
+    expr: 'anomaly_score > 1.0'
+    labels:
+      severity: warning
+    annotations:
+      summary: Anomaly Score exceeded 1.0. `rate(node_cpu_seconds_total)` is showing abnormal behavior. 
+```
+
+In the query expression we need to put a condition on the generated anomaly scores. Usually if the anomaly score is between 0.0 and 1.0, the analyzed value is not abnormal. The more anomaly score exceeded 1 the more our model is sure that value is an anomaly.
+You can choose your threshold value that you consider reasonable based on the anomaly score metric, generated by VmAnomaly. One of the best ways is to estimate it visually, by plotting the `anomaly_score` metric, along with predicted "expected" range of `yhat_lower` and `yhat_upper`. Later in this tutorial we will show an example
+____________________________________________
+## Docker Compose configuration
+Now we are going to configure the `docker-compose.yml` file to run all needed services.
+Here are all services we are going to run:
+
+![Docker compose services](guide-vmanomaly-docker-compose.png "Docker compose services")
```

Comment:
It's better to set the size for image by using html tags or reduce the size of images because they look huge on wide-angle screens

```
<p align="center">
  <img src="guide-vmanomaly-docker-compose.png" width="800" alt="Docker compose services">
</p>
```

Reply:
changed